generate.sh

```diff
@@ -28,54 +28,57 @@ INVENV=$(python -c 'import sys; print ("true" if hasattr(sys, "real_prefix") els
 # and change current directory to git repository.
 if [ "${WEBUI:-false}" == "true" ]; then
   BASE=$(pwd)
+  LOGFILE=${BASE}/temp/${EMAIL}/${UNIQUEID}.txt
   if ! [ "${SUBPROJECT}" == "" ]; then
     . ./multiple_generate.sh
     exit 0
   else
     mkdir -p temp/${EMAIL} && cd $_
-    echo -e "Cloning Repository...\n"
-    git clone -q https://:@${PLATFORM}/${USERNAME}/${REPONAME} "$UNIQUEID" >/dev/null 2>&1
+    echo -e "Cloning Repository...\n" | tee -a ${LOGFILE}
```

Comment:
Excellent! Although i'll make that in the morning. Time to sleep now :)
You may, however, carry on with reviewing the changes.